### ISSUE TITLE
NVDA Fails to recognise pop-up menus, tree views and other controls after installation of Microsoft Office

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5415]
**Reported by jscholes on 2015-10-13 23:57**
I recently posted about this on the NVDA development list in a decidedly unorganised fashion and erroneously claimed it was fixed while in fact the problem persists.

After installing Microsoft Office (so far we've tried both 2007 and 2010) on 64-bit Windows 7, NVDA simply stops recognising pop-up menus, menu bars, tree views, content in Mozilla Firefox, amongst others.  This makes the screen reader practically unusable.  As soon as MS Office is removed, everything goes back to normal.

I've attached a debug log so you can hopefully see what's going wrong.  JAWS seemingly remains uneffected, and the problem occurs with add-ons both enabled and disabled, and even with NVDA reset to factory defaults.

The repeated error in the debug log is a variation on the following:

DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (18:43:37):
oleacc.AccessibleObjectFromEvent with window 66380, objectID -4 and childID 0: [-2147312566](Error) Error loading type library/DLL

**Comment by leonardder** [comment]
This looks like another manifestation of #2807 and therefore I'm closing this as a duplicate. It will be fixed by #8075.
